### ISSUE TITLE
Proposer Slashing 2329

### ISSUE THREAD

**Issue by dankrad** [goerli/medalla#37]
Validator 2329 was slashed: https://medalla.beaconcha.in/validator/2329

Graffiti: Stefan#3917

**Comment by prestonvanloon** [comment]
This was Stefan#3917 in the Prysmatic Labs discord. I talked with them in a private message to understand what is going on. Here's the information I have collected.

Stefan was running 3 Prysm Validators with the same keystore, possibly connected to three different beacon nodes. They were testing out the experimental functionality for external slashing protect via their own central Prysm Slasher. I believe they incurred a race condition which the slasher did not adequately protect against or the slasher did not properly save the requests as possible slashing conditions. Ultimately they ended up creating a proposer slashing.
